src/Microsoft.DotNet.Build.Tasks.Packaging/src/GenerateNetStandardSupportTable.cs

```diff
@@ -68,27 +68,22 @@ public override bool Execute()
             SortedSet<Version> knownNetStandardVersions = new SortedSet<Version>();
             List<SupportRow> rows = new List<SupportRow>(Reports.Length);
 
-            foreach (var report in Reports.Select(r => r.GetMetadata("FullPath")))
+            foreach (var reportPath in Reports.Select(r => r.GetMetadata("FullPath")))
```

Comment:
The properties in this task aren't really documented at all.  It would be nice to know what a valid "Reports" item is and what metadata is available in case some change needs to be made in the future.